test/unit/data_hygiene/policy_tagger_test.rb

```diff
@@ -0,0 +1,81 @@
+require "test_helper"
+require "data_hygiene/policy_tagger"
+
+class PolicyTaggerTest < ActiveSupport::TestCase
+  setup do
+    @content_id_1 = SecureRandom.uuid
+    @content_id_2 = SecureRandom.uuid
+    @published_edition = create(:published_publication)
+    @document = @published_edition.document
+    @draft_edition = create(:draft_publication, document: @document)
+    stub_registration
+  end
+
+  test "updates policies for all editions of the document" do
+    # adding
+    policies_to_add = [@content_id_1, @content_id_2]
+    PolicyTagger.new(
+      slug:               @document.slug,
+      policies_to_remove: [],
```

Comment:
You guard that occurrence with your use of `fetch` earlier on. In fact, the other code in the ternary expression will never run if `fetch` fails as it doesn't return a truthy/falsy it raises an error instead. I'll comment on that.